core-services/prow/02_config/_plugins.yaml

```
@@ -770,6 +770,12 @@ config_updater:
         - ci
       gzip: true
       name: ci-operator-misc-configs
+    ci-operator/config/**/openshift-priv-insights-operator-stage.yaml:
```

Would the following be better?
```suggestion
    ci-operator/config/openshift-priv/openshift-priv-insights-operator-stage.yaml:
```